securityscanutils/trivy_scanner.go

```diff
@@ -104,6 +105,6 @@ func (t *TrivyScanner) executeScanWithRetries(ctx context.Context, scanArgs []st
 	return false, false, eris.Errorf("Trivy scan with args [%v] did not complete after %d attempts", scanArgs, t.scanMaxRetries)
 }
 
-func IsImageNotFoundErr(logs string) bool {
```

Comment:
Technically, since we are changing the signature of an exported method, this would make these changes breaking. It could be that this method was never consumed outside of this package, so it might not be an issue, I just wanted to call it out as something to be on the lookout moving forward

Reply:
I think that's a good callout. My assumption here was that only the `securityscanutils` package was using this function, but I technically can't guarantee that nothing outside `go-utils` is using this, so I'll revert this change.